include/swift/AST/Pattern.h

```diff
@@ -513,14 +513,13 @@ class EnumElementPattern : public Pattern {
   EnumElementPattern(TypeLoc ParentType, SourceLoc DotLoc, DeclNameLoc NameLoc,
                      DeclNameRef Name, EnumElementDecl *Element,
                      Pattern *SubPattern, Optional<bool> Implicit = None)
-    : Pattern(PatternKind::EnumElement),
-      ParentType(ParentType), DotLoc(DotLoc), NameLoc(NameLoc), Name(Name),
-      ElementDeclOrUnresolvedOriginalExpr(Element),
-      SubPattern(SubPattern) {
+      : Pattern(PatternKind::EnumElement), ParentType(ParentType),
```

Comment:
Nothing seems to have changed in this file except whitespace. Can you revert this out?